server/src/main/java/com/coincoinche/engine/CoincheGameRound.java

```diff
@@ -0,0 +1,51 @@
+package com.coincoinche.engine;
+
+import com.coincoinche.engine.game.GameResult;
+import com.coincoinche.engine.game.RedBlueRotatingPlayersGame;
+import com.coincoinche.engine.teams.Player;
+import com.coincoinche.engine.teams.Team;
+import java.util.List;
+
+/** Implementation of a coinche round. */
+public class CoincheGameRound extends RedBlueRotatingPlayersGame<Player> {
+
+  private GameState state;
+
+  public CoincheGameRound(Team redTeam, Team blueTeam) {
+    super(redTeam, blueTeam);
+  }
+
+  GameResult<Team> moveWasApplied() {
```

Comment:
Can you add a small comment on this method explaining what it does? i.e. if I understand well it's a callback after each move was applied.

Reply:
Yes, it's supposed to be called after a move was applied. I'll add a comment to explain this.